src/common/utils/stringHelpers.js

```diff
@@ -20,6 +20,10 @@ import {
   TYPE_WORD,
   TYPE_STORY,
   TYPE_SONG,
+  TYPE_MEDIA,
+  AUDIO,
```

Comment:
I would suggest adding `TYPE_AUDIO`, `TYPE_VIDEO`, and `TYPE_IMAGE` to our list of `searchParams` constants (like you did for `TYPE_MEDIA`). Even though the value of the constants you have used are currently the same, creating a searchParam constant will identify the specific use case of the string and allow for easier maintenance and future updates.